[DSM] PEPPER-882

pegahtah opened the pull request:
## Context

bash scripts to create the pubsub topics and set the events from buckets 

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

pegahtah committed:
PEPPER-882
Created a bash script for creating the initial and results topics in pubsub

pegahtah commented on cloud-functions/cf-file-scanner/dsm-cf-file-scanner-setup/dsm-setup.sh:
```diff
@@ -23,10 +23,28 @@ fi
+./create-bucket.sh "$project_name" "$study_name-uploaded-files"
 
-# Call the create_bucket.sh script to create the bucket
+## Call the create_bucket.sh script to create the bucket
```
unfortunately we can't check if the bucket exists because of public access prevention that we have set. This will try to create it and fail in the case that it already exists, however the rest of the script is not dependent on the success of the creation in this run, and can go on if the bucket was cerated previously .

otchet-broad replied:
Hmmm... Maybe we should investigate this.
gsutil ls gs://cmi-osteo2-files/ works for me.

So does 
gsutil ls gs://cmi-osteo2-uploaded-file

pegahtah committed:
Merge branch 'develop' into PEPPER-882

pegahtah committed:
PEPPER-882
Moved changes into a separate directory for DSM and also added a check for bucket pre-existing

pegahtah committed:
Merge branch 'develop' into PEPPER-882

pegahtah committed:
Merge remote-tracking branch 'origin/PEPPER-882' into PEPPER-882

pegahtah committed:
PEPPER-882
Changed the directory name
